integration_tests/test_dataset_download.py

```diff
@@ -0,0 +1,94 @@
+import os
+import unittest
+from typing import List
+
+from requests import HTTPError
+
+from kagglehub import dataset_download
+
+from .utils import create_test_cache
+
+HANDLE = "humanaaize/lung-cancer-dataset/versions/1"
+
+
+class TestDatasetDownload(unittest.TestCase):
+    def list_files_recursively(self, path: str) -> List[str]:
```

Comment:
can you move this to `integration_tests/utils.py`. The import for this method will be `from .utils import list_files_recursively`.

then, update `integration_tests/test_model_download.py` to use it too and delete the duplicated code.

Same for the `assert_files` method below.